src/components/otp-exercise/otp.js

```diff
@@ -1,86 +1,86 @@
-// import React, {useEffect, useState} from 'react';
-// import {TextInput, View, StyleSheet, StatusBar, Alert} from 'react-native';
+import React, {useEffect, useState} from 'react';
+import {TextInput, View, StyleSheet, StatusBar, Alert} from 'react-native';
 
-// const inputs = new Array(6).fill(0);
+const inputs = new Array(6).fill(0);
 
-// function OTP() {
-//   const [otpTextInput] = useState([]);
-//   const [otp, setOtp] = useState(Array(6).fill(''));
+function OTP() {
+  const [otpTextInput] = useState([]);
+  const [otp, setOtp] = useState(Array(6).fill(''));
 
-//   useEffect(() => {
-//     otpTextInput[0].focus();
-//   }, []);
+  useEffect(() => {
+    otpTextInput[0].focus();
+  }, []);
 
-//   const focusPrevious = (key, index) => {
-//     if (key === 'Backspace' && index !== 0) otpTextInput[index - 1].focus();
-//   };
+  const focusPrevious = (key, index) => {
```

Comment:
This method is bound to event. Lets keep this event specific. Name it accordingly

Reply:
done